ci/gpu/build.sh

```diff
@@ -69,8 +69,8 @@ conda install "rmm=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
               "rapids-notebook-env=$MINOR_VERSION.*"
 
 # https://docs.rapids.ai/maintainers/depmgmt/ 
-# conda remove -f rapids-build-env rapids-notebook-env
-# conda install "your-pkg=1.0.0"
+conda remove --force rapids-build-env rapids-notebook-env
+conda install "pandas>=1.0,<1.1.0a0"
```

Comment:
Once the `rapids-build-env` is updated (https://github.com/rapidsai/integration/pull/61) these two lines will need to be removed.